### DISCUSSION TITLE
Smart Seach - Select all or select multiple images possible?

### DISCUSSION THREAD

**Discussion by maxmar391** [immich-app/immich#13186] [Q&A]
Dear all,

First of all thanks to everyone that contributes to immich. I think it is a fantastic product and I like it a lot!

I have recently used the smart search more intensively and I could not figure out how to select all search results or multiple images of the search results. 

For selecting all, I have used CTRL+A, but this only selects any available text of the user interface.  

For selecting multiple, I have tried drawing a rectangle over the images I wanted to select or use SHIFT+Click to select a range. Both does not work. The rectangle does not appear at all. SHIFT+Click seems to work on the "Photos" page, but not for the smart search results.

Does anyone have some tips how to handle this challenge?

I would be glad about some ideas and answers.

**Comment by bo0tzz** [comment]
#9159

**Reply by maxmar391** [reply]
@bo0tzz thanks for the link. I have upvote the feature request. Is there a way to see if the request is considered?
And I guess that means there is currently no easy way to select multiple images at once. So let's cross fingers for the implementation of the feature.
